daemon/api.go

```diff
@@ -731,20 +731,25 @@ func (*licenseData) Error() string {
 	return "license agreement required"
 }
 
+type snapDownloadOptions struct {
```

Comment:
don't think Download is a great name for this set of options.  snapstate has something very similar but internal called RevisionOptions, I would follow the lead from there and call these snapRevisionOptions, include LeaveCohort as well.

Reply:
Thanks, fixed.